### PR TITLE
Fix debian logentries Agent

### PR DESCRIPTION
**Pull request by CostyaRegev**
Fix Debian logentries agent installation ...

### CODE REVIEW

**Comment by CostyaRegev** [comment]
@joeheung please review

File: attributes/default.rb [hunk 1]
```diff
@@ -15,3 +15,7 @@
 
 # PGP Key Server
 default['le']['pgp_key_server'] = 'pgp.mit.edu'
+
+# Debian Release
+
+default['le']['deb'] = 'trusty'
```
Comment (joeheung) [inline attributes/default.rb]: Should we not default this to node['lsb']['codename'] and let users override if required?

Comment (CostyaRegev) [inline attributes/default.rb]: we can fix it and then the users will be able to override , but the problem is that when the users will use Deb the system will not have values for  node['lsb']['codename']  , i checked it with chef-shell  so in this case the cookbook will fail...

Comment (joeheung) [inline attributes/default.rb]: I'd like to maintain as much backward compatibility as possible. We're not going to break anything new by setting ```node['le']['deb']``` to ```node['lsb']['codename']``` as that's the current behaviour however hardcoding ```node['le']['deb']``` to "Trusty" will possibly have breaking changes.

Comment (CostyaRegev) [inline attributes/default.rb]: what do you suggest ?

Comment (joeheung) [inline attributes/default.rb]: Sorry for the delay in getting back to you, I'd set the attribute to default to the following:

```ruby 
default['le']['deb'] =  node['lsb']['codename']
```

That way users can override if they wish and this doesnt change the current behavior.

Comment (CostyaRegev) [inline attributes/default.rb]: K, i have changed the code please review...


*~Costya Regev**,DevOps,Totango*


*Website:*http://www.totango.com | *Mobile:* (+972)545722572 | *Skype: *
costya.regev

**Comment by joeheung** [comment]
+1 LGTM!

**Pull request merged**